scripts/character_test.js

```diff
@@ -13,7 +13,9 @@ var goodChar;
 $(function () {
     // need to define this after the page has loaded so validCharacterPattern
     // is available
-    goodChar = RegExp(validCharacterPattern, "u");
```

Comment:
Not sure how long we will keep having to support IE, but is it worth looking into using babel to help alleviate these kinds of issues:
https://babeljs.io/docs/en/babel-plugin-transform-es2015-unicode-regex
We could transform similar to our less tool since we don't have a full build pipeline.

Reply:
Thanks, I'll look into it.

Reply:
IE 11 is the only available Microsoft browser for Windows 7 and Windows 8. Windows 7 went out of extended support in Jan 2020, so we can happily put that OS behind us. Windows 8 goes out of extended support in Jan 2023 -- so it will continue getting OS updates until that time. While I would love to deep-6 IE11 support, I don't feel that we can do that without a little more heads-up to our community.

All that to say that we should continue some level of IE11 support for the Unicode roll-out and then start messaging to our community (both at pgdp.net and  a heads-up to our other code users) that we are dropping IE11 support at a specific time.

Reply:
Can't we use XRegEx here? That would be a more forward-looking approach and we use it elsewhere.

Reply:
exactly, babel will allow us to use modern javascript with less IE issues. Also, for what it is worth, the new chromium edge is available for windows 7, 8, and 10 as well as mac and soon linux:
https://en.wikipedia.org/wiki/Microsoft_Edge#Anaheim_(2019%E2%80%93present)

Reply:
We are way too close to release date to bring in more middleware. I'm happy to consider that for after the roll-out, but I'd just rather drop IE11 in 6 months instead.